app-backend/api/src/main/scala/utils/QueryParameters.scala

```diff
@@ -69,4 +72,24 @@ trait QueryParametersCommon extends QueryParameterDeserializers {
   def shapeQueryParams = (
     orgQueryParams & userQueryParameters & timestampQueryParameters
   ).as(ShapeQueryParameters.apply _)
+
+  def searchQueryParams = parameters(
+    'searchPattern.as[String].?
```

Comment:
I think this should just be `search` as the meaning is clear and many APIs use the same terminology.